csi/controller_server.go

```diff
@@ -1245,7 +1248,7 @@ func (cs *ControllerServer) waitForBackupControllerSync(volumeName, snapshotName
 	if err != nil {
 		return nil, err
 	}
-	if backup.SnapshotCreated != "" {
+	if backup != nil && backup.SnapshotCreated != "" {
```

Comment:
Why can `cs.getBackup` return `nil error` and `nil backup` simutaneously?

Reply:
Modified as discussed. Thanks